server/routes/index.ts

```diff
@@ -57,7 +58,16 @@ export default function routes(services: Services): Router {
   return router
 }
 
-// Setup prisoner summary session for routes with prisonNumber param
+// Setup prisoner summary session for routes with prisonNumber param and check the prisoner is in the users caseloads
 function prisonerSummarySetup(router: Router, services: Services) {
   router.param('prisonNumber', retrievePrisonerSummary(services.prisonerSearchService))
+  router.param(
```

Comment:
For every route that has `prisonNumber` as a path parameter, right after we've retrieved the prisoner summary, do the caseload checks 👍